agent/agent.go

```diff
@@ -195,6 +195,7 @@ func (a *agent) fetch() ([]byte, error) {
 	if err != nil {
 		return nil, err
 	}
+	req.Header.Set("Authorization", q)
```

Comment:
If we're going to send a device ID as an authorization header, we can remove it from the URL (https://github.com/mainflux/license/blob/e8fcdcec76ee110a52110e73a9924b15214710e3/agent/agent.go#L193), but we'll also need to remove `:id` from the service endpoint (https://github.com/mainflux/license/blob/e8fcdcec76ee110a52110e73a9924b15214710e3/service/api/transport.go#L65) and use authorization header instead of ID here: https://github.com/mainflux/license/blob/e8fcdcec76ee110a52110e73a9924b15214710e3/service/api/endpoint.go#L94.